test/e2e/v1alpha2/run-e2e-experiment.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"os"
+	"time"
+
+	k8syaml "k8s.io/apimachinery/pkg/util/yaml"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	experimentsv1alpha2 "github.com/kubeflow/katib/pkg/api/operators/apis/experiment/v1alpha2"
+	"github.com/kubeflow/katib/pkg/util/v1alpha2/katibclient"
+)
+
+const (
+	timeout = 20 * time.Minute
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		log.Fatal("Experiment name is missing")
+	}
+	expName := os.Args[1]
+	b, err := ioutil.ReadFile(expName)
+	if err != nil {
+		log.Fatal("Error in reading file ", err)
+	}
+	exp := &experimentsv1alpha2.Experiment{}
+	buf := bytes.NewBufferString(string(b))
+	if err = k8syaml.NewYAMLOrJSONDecoder(buf, 1024).Decode(exp); err != nil {
+		log.Fatal("Yaml decode error ", err)
+	}
+	kclient, err := katibclient.NewClient(client.Options{})
+	if err != nil {
+		log.Fatal("NewClient for Katib failed: ", err)
+	}
+	var maxtrials int32 = 3
+	var paralleltrials int32 = 2
+	exp.Spec.MaxTrialCount = &maxtrials
+	exp.Spec.ParallelTrialCount = &paralleltrials
```

Comment:
We should set a metric here and make sure it is collected in the best trial.

Reply:
done